### ISSUE TITLE
[Question] Package Server (0.14.2) Is this the correct state? 

### ISSUE THREAD

**Issue by GrooveCS** [code-ready/crc#1332]
## General information

  * OS: **Windows**
  * Hypervisor: **Hyper-V** 
  * Did you run `crc setup` before starting it (Yes/No)? **YES**
  * Running CRC on: **Laptop**

### CRC version
```!bash
# Put `crc version` output here
crc version: 1.11.0+883ca49
OpenShift version: 4.4.5 (embedded in binary)
```
  
### CRC status
```!bash
# Put `crc status --log-level debug` output here
DEBU Running 'C:\Users\Developer\.crc\bin\oc\oc.exe get co -ojson --kubeconfig C:\Users\Developer\.crc\machines\crc\kubeconfig --context admin --cluster crc'
DEBU About to run SSH command:
df -B1 --output=size,used,target /sysroot | tail -1
DEBU SSH cmd err, output: <nil>: 32719437824 12208078848 /sysroot
CRC VM:          Running
OpenShift:       Running (v4.4.5)
Disk Usage:      12.21GB of 32.72GB (Inside the CRC VM)
Cache Usage:     40.95GB
Cache Directory: C:\Users\Developer\.crc\cache
```

## CRC config
```bash
# Put `crc config view` output here
```

## Host Operating System
```bash
# Put the output of `systeminfo` in case of Windows

Host Name:                 DESKTOP-TF4L8CF
OS Name:                   Microsoft Windows 10 Pro for Workstations
OS Version:                10.0.18363 N/A Build 18363
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          Developer
Registered Organization:   
Product ID:                00391-90050-00000-AA002
Original Install Date:     3/26/2020, 3:23:44 PM
System Boot Time:          6/12/2020, 5:19:30 PM
System Manufacturer:       Acer
System Model:              Aspire TC-865
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 158 Stepping 10 GenuineIntel ~2808 Mhz
BIOS Version:              American Megatrends Inc R01-A3, 5/16/2018
Windows Directory:         C:\Windows
System Directory:          C:\Windows\system32
Boot Device:               \Device\HarddiskVolume3
System Locale:             en-us;English (United States)
Input Locale:              N/A
Time Zone:                 (UTC-05:00) Eastern Time (US & Canada)
Total Physical Memory:     65,423 MB
Available Physical Memory: 44,535 MB
Virtual Memory: Max Size:  75,151 MB
Virtual Memory: Available: 53,862 MB
Virtual Memory: In Use:    21,289 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\DESKTOP-TF4L8CF
Hotfix(s):                 11 Hotfix(s) Installed.
                           [01]: KB4552931
                           [02]: KB4513661
                           [03]: KB4516115
                           [04]: KB4517245
                           [05]: KB4528759
                           [06]: KB4537759
                           [07]: KB4541338
                           [08]: KB4552152
                           [09]: KB4560959
                           [10]: KB4561600
                           [11]: KB4560960
Network Card(s):           4 NIC(s) Installed.
                           [01]: Intel(R) Wireless-AC 9462
                                 Connection Name: Wi-Fi
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.1.1
                                 IP address(es)
                                 [01]: 192.168.1.114
                                 [02]: fe80::d5ce:74b:f58:55a3
                           [02]: Realtek PCIe GbE Family Controller
                                 Connection Name: Ethernet
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.1.1
                                 IP address(es)
                                 [01]: 192.168.1.113
                                 [02]: fe80::d8e4:a052:4f11:d168
                           [03]: Bluetooth Device (Personal Area Network)
                                 Connection Name: Bluetooth Network Connection
                                 Status:          Media disconnected
                           [04]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (Default Switch)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.9.193
                                 [02]: fe80::534:ff05:7329:2dd3
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.
```

### Steps to reproduce

  1. 
  2. 
  3. 
  4. 

### Expected

See no error message for **Status**.

### Actual

See error message for **Status** `Cannot update Catalog source was removed`.
### Installed Operators: _Package Server (0.14.2)_
![Installed Operator](https://qas-shared-assets.s3.amazonaws.com/snaps/9zxovdxu66e)

### Logs

Here is a copy of the log from today: [crc.log](https://github.com/code-ready/crc/files/4815321/crc.log) and here copy at [Gist #file-2020-06-22t1553-crc.log](https://gist.github.com/GrooveCS/8f2495ce507800814dbda8833f7b23fe#file-2020-06-22t1553-crc-log)

```!powershell
┌[Parris@com.RedHat]
└[~\CRC\1.11]> .\crc status --log-level debug
DEBU Running 'C:\Users\Developer\.crc\bin\oc\oc.exe get co -ojson --kubeconfig C:\Users\Developer\.crc\machines\crc\kubeconfig --context admin --cluster crc'
DEBU About to run SSH command:
df -B1 --output=size,used,target /sysroot | tail -1
DEBU SSH cmd err, output: <nil>: 32719437824 16335499264 /sysroot
CRC VM:          Running
OpenShift:       Running (v4.4.5)
Disk Usage:      16.34GB of 32.72GB (Inside the CRC VM)
Cache Usage:     40.95GB
Cache Directory: C:\Users\Developer\.crc\cache
```

**Comment by anjannath** [comment]
@GrooveCS Nope, this should not be the state, i tried with bundle 4.4.8 (which is to be release today) and did not see this. It could be that the operator was reconciling and needed more time to be in a ready state!

![Capture11](https://user-images.githubusercontent.com/8885742/85382560-b6665580-b55c-11ea-8f2e-afac2aa30049.PNG)

**Comment by GrooveCS** [comment]
@anjannath 

I started the update process yesterday and going to check today to see if resolved by updating to `CRC 4.4.8` on winOS lab workstation.

**Comment by GrooveCS** [comment]
@anjannath

Uninstalled `CRC 1.11` and installed `CRC 1.12` and everything seems to work a lot better now. 

**Installed Operator - Package Server Operator**
![Package Server - Succeed State ](https://qas-shared-assets.s3.amazonaws.com/snaps/o7hk43jjuu)

**Operator Hub Availability** 
![Operator Hub - Available Promptly](https://qas-shared-assets.s3.amazonaws.com/snaps/oigood61j2b)
